src/lib/config-toml.ts

```
@@ -0,0 +1,38 @@
+import * as smolToml from 'smol-toml';
+import { validateConfig, type InsforgeConfig } from './config-schema.js';
+
+export function parseConfigToml(input: string): InsforgeConfig {
+  let parsed: unknown;
+  try {
+    parsed = smolToml.parse(input);
+  } catch (err) {
+    throw new Error(`TOML parse error: ${(err as Error).message}`);
```

ESLint

[error] There is no `cause` attached to the symptom error being thrown.

(preserve-caught-error)